From d87b1a298838bf8195b523f875dd31c35cb7e5bb Mon Sep 17 00:00:00 2001
From: ChonDoe <thphantomblog@gmail.com>
Date: Tue, 21 Feb 2023 11:17:20 +0000
Subject: [PATCH] Remove Useless things v2

---
 config/common_mobile.mk |  9 ---------
 1 files changed, 9 deletions(-)

diff --git a/config/common_mobile.mk b/config/common_mobile.mk
index 98be187e..c27d0d41 100644
--- a/config/common_mobile.mk
+++ b/config/common_mobile.mk
@@ -14,20 +14,11 @@ PRODUCT_PRODUCT_PROPERTIES += \
 
 # Apps
 PRODUCT_PACKAGES += \
-    Aperture \
     Backgrounds \
-    Eleven \
-    Etar \
     ExactCalculator \
-    Jelly \
     Profiles \
     Seedvault
 
-ifneq ($(TARGET_EXCLUDES_AUDIOFX),true)
-PRODUCT_PACKAGES += \
-    AudioFX
-endif
-
 # Charger
 PRODUCT_PACKAGES += \
     charger_res_images
-- 
2.34.1

