From a6de728045763e522d40a013a0d86a4cd7bc57f8 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Wed, 22 Feb 2023 14:34:48 +0000
Subject: [PATCH 2/2] Update TurboAdapter sepolicy

---
 sepolicy/turbo_adapter.te | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/sepolicy/turbo_adapter.te b/sepolicy/turbo_adapter.te
index 7e8ed2e..c49cdfa 100644
--- a/sepolicy/turbo_adapter.te
+++ b/sepolicy/turbo_adapter.te
@@ -2,7 +2,7 @@
 # this means that TurboAdapter doesn't get the platform_app permissions any more, so we need to
 # list everything that it needs here.
 
-type turbo_adapter, domain, coredomain;
+type turbo_adapter, domain, coredomain, system_suspend_internal_server;
 
 app_domain(turbo_adapter)
 
@@ -10,5 +10,15 @@ app_domain(turbo_adapter)
 allow turbo_adapter app_api_service:service_manager find;
 allow turbo_adapter hal_turbo_adapter_hwservice:hwservice_manager find;
 
+# PAS: for PowerStatsHalDataProvider
+hal_client_domain(turbo_adapter, hal_power_stats)
+
+# PAS: for GoogleCpuTimeProvider
+r_dir_file(turbo_adapter, proc_uid_cputime_showstat);
+
+# PAS: for SuspendControlServiceDataProvider
+binder_call(turbo_adapter, system_suspend_internal_server)
+get_prop(turbo_adapter, suspend_prop)
+
 # To find and call hal_power_default so turbo can obtain the service extension (IPowerExt)
 hal_client_domain(turbo_adapter, hal_power)
-- 
2.34.1

