From cb60627144a0296f55b65bd6cd071527f8332b9a Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Fri, 7 Apr 2023 02:05:26 +0000
Subject: [PATCH] Nuke duplicated apns-conf.xml and fix Dynamic Root

---
 base.mk      | 4 ----
 phh-root.sh  | 2 ++
 3 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/base.mk b/base.mk
index 6db2925..62c27a0 100644
--- a/base.mk
+++ b/base.mk
@@ -8,10 +8,6 @@ PRODUCT_COPY_FILES := \
 	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
 
-#Use a more decent APN config
-PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
-
 BOARD_PLAT_PRIVATE_SEPOLICY_DIR += device/phh/treble/sepolicy
 PRODUCT_PACKAGE_OVERLAYS += device/phh/treble/overlay
 
diff --git a/phh-root.sh b/phh-root.sh
index 8be3e4e..69848c0 100644
--- a/phh-root.sh
+++ b/phh-root.sh
@@ -33,6 +33,8 @@ phhnoroot() {
 
 if [ "$1" == "persist.sys.phh.dynamic_superuser" ]; then
 	if [[ "$prop_value" != "0" && "$prop_value" != "1" ]] || [ -e /sbin/magisk ]; then
+		stop sudaemon
+		rm -rf /data/su || true
 		exit 1
 	fi
 
-- 
2.34.1

